### ISSUE TITLE
Set directory structure and filename specifications

### ISSUE THREAD

**Issue by mekline** [psych-ds/psych-DS#9]
(These have been dormant for a little, but are unfinished!)

Useful contributions: 

Over on [the draft](https://docs.google.com/document/d/1u8o5jnWk0Iqp_J06PTu5NjBfVsdoPbBhstht6W0fFp0/edit#), clean up some of the existing comments/migrate anything we need to keep in mind to a comment on this issue. 

Flesh out the proposed naming conventions, or ask questions about what's unclear.

**Comment by mekline** [comment]
Discussion about the naming schemes is also happening on the google group:

https://groups.google.com/forum/#!topic/psych-data-standards/1kW53zHbUbw

**Comment by mekline** [comment]
We've made a bunch of progress in the thread! I updated the [draft](https://docs.google.com/document/d/1u8o5jnWk0Iqp_J06PTu5NjBfVsdoPbBhstht6W0fFp0/edit#heading=h.1m6sa5bs9j0e) - Check out the section called 'File Naming.' I've done my best attempt at a consolidation, as well as giving the specification a general run-through to check it matches other conversations on the google group.  In particular, I'm testing out the proposed regular expression for filenames that would/wouldn't be allowed, and ask a number of questions as comments on the doc! 

Then, because some of the keywords are also liable to appear as columns inside the datasheet, I added RECOMMEND-level to the specification that the keywords also be used as column names in TSVs (ie that if you have an experiment column, name it 'study', etc.) Is this too fine grained? We can drop it IFF we think it's more confusing than helpful at this point (I can see a V2 of the spec providing much more guidance on column names, but we're not there yet I think?)
